components/hab/src/cli.rs

```diff
@@ -27,6 +27,9 @@ use url::Url;
 use command::studio;
 use feat;
 
+static RING_ENVVAR: &'static str = "HAB_RING";
+static RING_KEY_ENVVAR: &'static str = "HAB_RING_KEY";
```

Comment:
Looks like the documentation should also be tweaked [here](https://github.com/habitat-sh/habitat/blob/46fc6ad232bad6a20e2a834251e2e3ac224645b8/www/source/partials/docs/_reference-environment-vars.html.md.erb#L24).